### ISSUE TITLE
Unable to access the CDN path for png icons for version 3.4.0  "https://unpkg.com/@tabler/icons-png@3.4.0/icons/home.png" [BUG] 

### ISSUE THREAD

**Issue by G-Varada** [tabler/tabler#1895]
### Browser

On Crome and Edge

### OS

Windows 11

### Screen size

1920x1080

### Describe the bug

1. Unable to access the CDN path for Tabler png icons for version 3.4.0  "https://unpkg.com/@tabler/icons-png@3.4.0/icons/home.png"

2. With "npm install  @tabler/icons-png", "icons" subdirectory was not created and hence png icons files are not downloaded and available to use.

### How to reproduce

1. Unable to access the CDN path for Tabler png icons for version 3.4.0  "https://unpkg.com/@tabler/icons-png@3.4.0/icons/home.png"

2. With "npm install  @tabler/icons-png", "icons" subdirectory was not created and hence png icons files are not downloaded and available to use.

### Screenshots

![Screenshot 2024-05-17 170047](https://github.com/tabler/tabler/assets/139129172/1410642f-f045-4dc7-b1bd-6809961cc39a)
![Screenshot 2024-05-17 170911](https://github.com/tabler/tabler/assets/139129172/53fbd656-863c-428b-9719-728e7c8d4053)

**Comment by mmontone** [comment]
Yes. CDN mentioned here doesn't work: https://tabler.io/docs/icons/webfont

**Comment by BG-Software-BG** [comment]
The CDN links are fixed. 

Use https://unpkg.com/@tabler/icons-png@latest/icons/outline/home.png for outline version or https://unpkg.com/@tabler/icons-png@latest/icons/filled/home.png for filled version.

For webfont use https://cdn.jsdelivr.net/npm/@tabler/icons-webfont@latest/dist/tabler-icons.min.css.

The docs will be updated soon.
